Library/Formula/sane-backends.rb

```diff
@@ -45,17 +43,70 @@ def install
 end
 
 __END__
-diff --git a/include/sane/sane.h.orig b/include/sane/sane.h
-index 5320b4a..6cb7090 100644
---- a/include/sane/sane.h.orig
+https://lists.alioth.debian.org/pipermail/sane-devel/2015-October/033972.html
```

Comment:
Could you move this URL to above the `patch :DATA` line and add a quick comment explaining what the patch is for? Thanks!